BenadrylAutomation/Configurations/config.properties

```diff
@@ -1,5 +1,5 @@
-stage_url=http://test.benadryl.com/
+stage_url=http://app-23191.on-aptible.com/
 live_url=https://www.benadryl.com/
 browserName=Chrome
-headlessChrome=yes
+headlessChrome=No
```

Comment:
@zahinSJI why this value is set no??? It should be set always yes before creating PR

Reply:
Change applied

Reply:
@zahinSJI cannot see the change